src/app/public/modules/action-button/action-button.component.ts

```diff
@@ -15,9 +15,15 @@ import {
   templateUrl: './action-button.component.html'
 })
 export class SkyActionButtonComponent {
```

Comment:
Should we add a description up above for the component? It's probably not necessary, but just wondering if we should have it for consistency since the other components in the module all have descriptions. I'd think it would just be something condensed version of the page summary like `Creates a button to present users with an option to move forward with tasks.`